deployment/k8s/charts/aggregates/migrationAssistant/values.yaml

```diff
@@ -19,24 +20,37 @@ capture-proxy:
       yamlPath: "allow_insecure"
 
 migration-console:
+  snapshotVolumeEnabled: true
+  snapshotVolumePvc: "snapshot-volume-pvc"
   parameters:
     nothingImportant:
       value: false
 
 bulk-document-loader:
+  snapshotVolumeEnabled: true
+  snapshotVolumePvc: "snapshot-volume-pvc"
   parameters:
     luceneDir:
       value: /tmp
       allowRuntimeOverride: true
     snapshotName:
-      value: rfs
+      value: rfs-snapshot
       allowRuntimeOverride: true
     targetHost:
-      value: https://opensearch-cluster-master.mcc:9200/
+      value: https://opensearch-cluster-master:9200
+      allowRuntimeOverride: true
+    targetUsername:
+      value: admin
+      allowRuntimeOverride: true
+    targetPassword:
+      value: myStrongPassword123!
       allowRuntimeOverride: true
```

Comment:
once the shared config stuff merges in, it would be good to put these into shared configs.  Longer term, it will be cool to have separate targets in the shared config - like a targets map for each of the different target migrations that we're testing (then select out which one we want).  We need some migration console support for that UX first though.